src/themes/default.css

```diff
@@ -4,14 +4,30 @@
   /* Colors */
   --gr-color-primary: #f27a1a;
   --gr-color-primary-hover: #ef6114;
+  --gr-color-primary-outline: #FFF;
   --gr-color-secondary: #273142;
   --gr-color-secondary-hover: #0f131a;
-  --gr-color-success: #0bc15c;
-  --gr-color-success-hover: #09a44e;
-  --gr-color-content-primary: #273142;
-  --gr-color-content-secondary: #6e7787;
-  --gr-color-content-tertiary: #95a1b5;
-  --gr-color-content-passive: #afbbca;
+  --gr-color-secondary-outline: #FFF;
+  --gr-color-tertiary: #F1F2F7;
+  --gr-color-tertiary-hover: #D5D9E1;
+  --gr-color-tertiary-outline: #FFF;
+  --gr-color-success: #0BC15C;
+  --gr-color-success-hover: #09A44E;
+  --gr-color-success-outline: #FFF;
+  --gr-color-error: #FF5043;
+  --gr-color-error-hover: #FF3028;
+  --gr-color-error-outline: #FFF;
+  --gr-color-content-primary: #FFF;
+  --gr-color-content-outline-primary: #f27a1a;
```

Comment:
Outline font colors should be always same with the main color of the variant. So I think we should not need those outline color variables.

Reply:
Known issue. It will be solved in the next commit 👍. PR will be ready for review on Monday.